### DISCUSSION TITLE
My application is not getting approved

### DISCUSSION THREAD

**Discussion by Segiroo** [community/community#113555] [GitHub Education]
### Hi everyone,

I'm trying to apply for the educational pack but when I tried the first time I didn't have de 2FA and now  that I have the 2FA I can't upload the proof because it says that has been used previously.
![image](https://github.com/community/community/assets/90848945/ea5c8b73-0135-4750-bc61-f3dfb25b8c13)

**Comment by samus-aran** [comment]
Hi @al415421sergio 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
